README.md

```diff
@@ -672,6 +672,14 @@ Circus Train can be extended in various ways. This is an advanced feature that t
 * `LocationManagers` look after the source and replica locations and are an ideal point to implement both snapshot isolation and retired data clean-up.
 * `CompositeCopierFactory` allows the provision of multiple copiers for the same table in order to add functionality to the copy process, e.g. introduce compaction, copy to multiple destinations, etc.
 
+### External Extensions
+
+#### Circus Train DataSqueeze Copier
+The [DataSqueeze](https://github.com/ExpediaInceCommercePlatform/circus-train-datasqueeze) extension provides new Copiers for Circus Train which compact the data being replicated in various ways
```

Comment:
Looks good, last change request (I promise!) - put a full stop at the end of both sentences ;)

Reply:
Sure :-). It's done